tests/framework/extensions/charts/ranchermonitoring.go

```diff
@@ -0,0 +1,197 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rancher/rancher/pkg/api/steve/catalog/types"
+	catalogv1 "github.com/rancher/rancher/pkg/apis/catalog.cattle.io/v1"
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/pkg/wait"
+	"github.com/rancher/rancher/tests/integration/pkg/defaults"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+const (
+	RancherMonitoringNamespace    = "cattle-monitoring-system"
+	RancherMonitoringName         = "rancher-monitoring"
+	RancherMonitoringCRDName      = "rancher-monitoring-crd"
+	RancherMonitoringConfigSecret = "alertmanager-rancher-monitoring-alertmanager"
+)
+
+//InstallRancherMonitoringChart is a helper function that installs the rancher-montitoring chart.
+func InstallRancherMonitoringChart(client *rancher.Client, projectID, clusterName, version string) error {
```

Comment:
Out of curiosity is there a particular reason we picked monitoring to be out test chart? it is one of the heavier ones.  Something like rancher backup or cis would be much faster to test with.

Reply:
It was one of the charts that Brandon suggest to me to try, also it has been implemented in one of our python validation tests. No other reason besides that.